src/App.js

```diff
@@ -2,9 +2,9 @@ import './App.css';
 import React from 'react';
 import { Routes, Route } from 'react-router-dom';
 import BookList from './components/BookList';
-import NavBar from './components/navBar';
+import NavBar from './components/NavBar';
 import Categories from './components/categories';
-import Footer from './components/footer';
+import Footer from './components/Footer';
```

Comment:
- [ ] As requested by the previous reviewer, Kindly make sure to rename your jsx files name,  instead use capitalize eg: BookList.js, Footer.js, ... 😉 ✍🏻